lib/elixir/test/elixir/macro_test.exs

```diff
@@ -141,11 +141,26 @@ defmodule MacroTest do
       assert Macro.escape({:quote, [], [[do: :foo]]}) == {:{}, [], [:quote, [], [[do: :foo]]]}
     end
 
-    test "inspects container when a reference cannot be escaped" do
-      assert_raise ArgumentError, ~r"~r/foo/ contains a reference", fn ->
-        Macro.escape(%{~r/foo/ | re_pattern: {:re_pattern, 0, 0, 0, make_ref()}})
+    test "escape container when a reference cannot be escaped" do
+      assert_raise ArgumentError, ~r"contains a reference", fn ->
+        Macro.escape(%{re_pattern: {:re_pattern, 0, 0, 0, make_ref()}})
       end
     end
+
+    @tag skip: not function_exported?(:re, :import, 1)
```

Comment:
Let's use exclude in the test/test_helper.exs instead, for consistency? We should probably do `Code.ensure_loaded?(:re)` as well...

Reply:
I would add one called `re_import_exclude`, yes, so we have all of them handled in a single place instead of scattered.